packages/link-bins/src/index.ts

```diff
@@ -122,7 +127,9 @@ async function getPackageBins (
     return []
   }
 
-  if (R.isEmpty(pkg.bin) && !target.includes('node_modules')) {
+  const fromModules = await isFromModules(target)
+
+  if (R.isEmpty(pkg.bin) && !fromModules) {
```

Comment:
This solution is bad because you make a filesystem operation even if the bin is not empty. The following version is better
```suggestion
  if (R.isEmpty(pkg.bin) && !await isFromModules(target)) {
```